docs/components/compact.md

```diff
@@ -55,18 +55,27 @@ In fact, downsampling doesn't save you any space but instead it adds 2 more bloc
 
 ## Groups
 
-The compactor groups blocks using the external_labels added by the Prometheus who produced the block.
-The labels must be both _unique_ and _persistent_ across different Prometheus instances.
+The compactor groups blocks using the external*labels added by the Prometheus who produced the block.
```

Comment:
Those changes are weird.. do we need them?

Reply:
was added by vscode formatting, have reverted them